src/main/scala/intellij/haskell/settings/HaskellConfigurable.scala

```diff
@@ -26,9 +26,14 @@ import javax.swing.event.DocumentEvent
 class HaskellConfigurable extends Configurable {
   private var isModifiedByUser = false
   private val hlintOptionsField = new JTextField
+  private val useSystemGhcToggle = new JCheckBox
```

Comment:
Changing `useSystemGhcToggle` will not set `isModifiedByUser` flag.

Reply:
Will fix